selfie.c

```diff
@@ -2308,13 +2310,35 @@ void init_system() {
   if (is_boot_level_zero()) {
     BOOTLEVELZERO = 1;
 
-    // caution: the name of the executable must not have an extension to make this work
-    // try opening executable with zeroed flags which likely fails but just on Windows
-    if (signed_less_than(sign_extend(open(selfie_name, 0, 0), SYSCALL_BITWIDTH), 0))
+    // caution: this could also be true for executables on Linux/MacOs
```

Comment:
We should probably include a tiny bit of information on the system selfie detected here to be sure it works:

./selfie: this is the selfie system from selfie.cs.uni-salzburg.at with
./selfie: 64-bit unsigned integers and 64-bit pointers on boot level 0